core/src/main/scala/chisel3/experimental/hierarchy/core/Hierarchy.scala

```diff
@@ -103,6 +103,10 @@ object Hierarchy {
     def toTarget: IsModule = i match {
       case d: Definition[T] => new Definition.DefinitionBaseModuleExtensions(d).toTarget
       case i: Instance[T]   => new Instance.InstanceBaseModuleExtensions(i).toTarget
+      case _ =>
+        throw new Exception(
+          s"Internal Error: Please file an issue at https://github.com/chipsalliance/chisel3/issues: Match error: toTarget i=$i"
+        )
```

Comment:
This is the warning
```
[warn] /Users/chickm/SiFive/code/chisel3-cleanup-warnings-1/core/src/main/scala/chisel3/experimental/hierarchy/core/Hierarchy.scala:115:38: match may not be exhaustive.
[warn] It would fail on the following input: (_ : chisel3.experimental.hierarchy.core.SealedHierarchy[?])
[warn]     def toAbsoluteTarget: IsModule = i match {
[warn]                                      ^
```

Reply:
will leave expression because casing SealedHierarchy will not really help problem